### DISCUSSION TITLE
SMTP send error Failure sending mail. Unable to read data from the transport connection: The connection was closed.

### DISCUSSION THREAD

**Discussion by nitss27** [postalserver/postal#2764] [Help with using Postal]
i face  trouble in sending email test, shows me error on smtper,net, can someone please help me to fix this?

SMTP send error
Failure sending mail.
Unable to read data from the transport connection: The connection was closed.

![image](https://github.com/postalserver/postal/assets/157808078/1f16ddc5-f4b8-4ccd-8ba5-57b6fff28d72)

**Comment by logicdelta** [comment]
@nitss27  I am also facing the exact error.  Also, domain:5000/users/new form is also not saving the detail.

**Comment by nitss27** [comment]
Have you found any solution?

**Reply by logicdelta** [reply]
@nitss27  yes , I got 2 errors.
1) Rabbitmq was not working. that is fixed now.
2) FrozenError: can't modify frozen String: "" . Still working on it.

**Comment by nitss27** [comment]
Can u please help me, this is new for me I don't know how to perform it
